### PR TITLE
Remove bootstrap css

### PR DESCRIPTION
**Pull request by ficocelliguy**
Remove bootstrap, reorganize files and clean css

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.table {
+    width: auto;
+    margin-bottom: auto;
+    background-color: transparent;
+    border-spacing: 0;
+    border-collapse: collapse;
+}
+
```
Comment (PsychicMuffin) [inline css/style.css:40]: extra space

File: css/damageTable.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
+.damageTable {
+    position: fixed;
+    right: 10px;
+    top: 90px;
+    width: 17em;
+}
+
+.damageTable .control-label {
```
Comment (PsychicMuffin) [inline css/damageTable.css]: .control-label was the bootstrap name, we should probably rename this to something more specific, or if we can't think of anything just "label"

File: css/damageTable.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.damageTable .damageInput,
+.damageTable .damageOutput {
+    padding: 0;
+}
+
+.damageTable .damageOutput {
+    padding-left: 5px;
+}
```
Comment (PsychicMuffin) [inline css/damageTable.css]: This should go in the block below right?

File: css/input.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+.input {
+    font-size: 12px;
+    line-height: 1.5;
+    border-radius: 3px;
+    display: inline-block;
+    padding: 5px;
+    color: #555;
+    background-color: #fff;
+    background-image: none;
```
Comment (PsychicMuffin) [inline css/input.css]: background can be condensed to "background: #fff none;"

File: css/input.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    border: 1px solid #ccc;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+    -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+    transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
```
Comment (PsychicMuffin) [inline css/input.css]: spaces after the commas in lines 38-42

File: css/input.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    border-bottom-left-radius: 0;
+}
+
+/* Prevent increment/decrement arrows from appearing on number fields*/
+.number.noIncrement::-webkit-outer-spin-button,
+.number.noIncrement::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+}
+
+.number.noIncrement{
```
Comment (PsychicMuffin) [inline css/input.css]: space before brace

File: css/input.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    display: table-cell;
+}
+
+.input-group .input-group-addon.flat:hover {
+    background-color: #C2CED7;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -o-user-select: none;
```
Comment (PsychicMuffin) [inline css/input.css]: why does webstorm not recognize this property?

File: css/input.css [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+    -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+    -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+    transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+}
+
+.input:focus {
+    border-color: #66afe9;
+    outline: 0;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px rgba(102,175,233,.6);
```
Comment (PsychicMuffin) [inline css/input.css]: spaces after all the commas

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+h4 small {
+    font-size: 75%;
+    font-weight: 400;
+    line-height: 1;
+    color: #777;
+}
+
+body {
+    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
```
Comment (PsychicMuffin) [inline css/style.css]: spaces after commas

File: css/input.css [hunk 10]
```diff
@@ -0,0 +1,112 @@
+#armor {
```
Comment (PsychicMuffin) [inline css/input.css:1]: the armor and endurance charges look kinda funny with their vertical alignment now no?

**Review by PsychicMuffin — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+    border-color: #d43f3a;
+}
+
+.btn-danger:hover {
+    color: #fff;
+    background-color: #c9302c;
+    border-color: #ac2925;
+}
+
+.btn {
```
Comment (PsychicMuffin) [inline css/style.css]: can we just expand this to be "button", it was always weird they abbreviated it to begin with, and we don't use other abbreviations really.

File: css/style.css [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+    width: 30px;
+}
+
+.section .btn {
+    font-weight: bold;
+    opacity: 0;
+    transition: opacity .35s ease;
+}
+
+.btn-danger {
```
Comment (PsychicMuffin) [inline css/style.css]: we can probably rename these to something more specific to our use, like "delete"

File: css/style.css [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    background-image: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    overflow: visible;
+}
+
+.tableContainer tr:hover .btn {
```
Comment (PsychicMuffin) [inline css/style.css]: I think it's a bit odd that some of our css uses hyphens and some uses camel case now

File: css/input.css (hunk 7 above)
Comment (ficocelliguy) [reply css/input.css]: It got removed from opera recently, apparently. The webkit part should work.

File: css/input.css (hunk 10 above)
Comment (ficocelliguy) [reply css/input.css:1]: I fixed them now.

File: css/input.css [hunk 14]
```diff
@@ -0,0 +1,110 @@
...
+    font-size: 12px;
+    line-height: 1.5;
+    border-radius: 3px;
+    display: inline-block;
+    padding: 5px;
+    color: #555;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
```
Comment (PsychicMuffin) [inline css/input.css]: spaces after commas these 2 lines

**Review by PsychicMuffin — CHANGES_REQUESTED** [review]

**Review by PsychicMuffin — APPROVED** [review]

**Pull request merged**